### ISSUE TITLE
Add Table::truncate() && AdapterInterface::truncateTable() methods

### ISSUE THREAD

**Issue by samizdam** [robmorgan/phinx#703]
According to the wiki: It was officially introduced in the SQL:2008 standard. 
And supported by all main database vendors / drivers. 
Do Phinx need this feature?

**Comment by robmorgan** [comment]
Phinx only needs this feature if it turns out to be increasingly popular amongst users. It might come in handy with Seeders, but i'll leave it for now.

**Comment by WillGibson** [comment]
@robmorgan I just found myself here because I need to truncate a table in a migration too. I'll hand roll mine for now, but could submit a PR for this feature if you wanted to reopen the issue?

**Comment by rickbolton** [comment]
@robmorgan nice work BTW. I'm here also as I need to truncate before running some seeders.

**Comment by robmorgan** [comment]
@WillGibson sure, feel free to contribute something!

**Comment by andtxr** [comment]
There is an active PR for this?

**Comment by harkenn** [comment]
Still waiting for this.

**Comment by Techworker** [comment]
For those stumbling upon this issue (while this MR is not accepted), use this:

It's essentially what the MR from @edsonmedina does.

```php
<?php
/* ... seed stuff ... */

$tableName = 'seed_table_name';
$table = $this->table($tableName);

// truncate table
$table->getAdapter()->execute(
    'TRUNCATE TABLE ' . $table->getAdapter()->quoteTableName($tableName)
);

// seed..
$table->insert($data)->save();
```

**Comment by piotr-cz** [comment]
There is now a PR https://github.com/robmorgan/phinx/pull/978 for this feature.

**Comment by robmorgan** [comment]
Merged to `master` in a10918dbe6da3a5b19be1b89f0953d82b0adcb98
